domains/pallets/domain-registry/src/tests.rs

```diff
@@ -122,6 +134,7 @@ impl pallet_domain_registry::Config for Test {
     type MinDomainOperatorStake = MinDomainOperatorStake;
     type MaximumReceiptDrift = MaximumReceiptDrift;
     type ReceiptsPruningDepth = ReceiptsPruningDepth;
+    type CoreDomainTracker = DomainTracker;
```

Comment:
yes, but this was intentional :)

Reply:
Why mock it when there is already an impl. So I usually end up connecting the pallet if already make sense.

Reply:
Okay, just a personal taste, prefer less code if possible, 10 lines saved, and one less dependency in this case. Also an empty `()` does not always mean to mock, although it's often used in the test env for easier setup, it can be also used in some production contexts as it's a proper implementation which indicates just doing nothing.

Reply:
Less code here is more code to impl for ()
I like the explicitness of the deps because it always check the actual implementation.
I think we won't save any code anywhere. But sure it might not mean mock. I personally prefer this since it gives me prod context on deps as well. Anyway I the lines of code are moved somewhere else.